src/main/java/view/BouquetPrinter.java

```diff
@@ -0,0 +1,12 @@
+package view;
+
+import model.Bouquet;
+
+public class BouquetPrinter {
+
+    public void print(Bouquet bouquet) {
+
+        System.out.println(bouquet.toString());
```

Comment:
I would have print everything individually, since to string is a service method (not for user usually)